barretenberg/cpp/src/barretenberg/translator_vm/translator_circuit_builder.hpp

```diff
@@ -96,53 +96,58 @@ class TranslatorCircuitBuilder : public CircuitBuilderBase<bb::fr> {
         X_LOW_Y_HI,
         X_HIGH_Z_1,
         Y_LOW_Z_2,
-        P_X_LOW_LIMBS,                    // P.xₗₒ split into 2 68 bit limbs
```

Comment:
had to sort the entities differently to get a contiguous range of repeating commitments